<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DtIntervalWidget</class>
 <widget class="QWidget" name="DtIntervalWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>115</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="6">
      <widget class="IntervalSlider" name="widgetSelectInterval" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="2">
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>Интервал прогноза</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" rowspan="2" colspan="2">
      <widget class="QLabel" name="betweenDateLabel">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4" rowspan="2">
      <widget class="QToolButton" name="toolButtonToDt">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/icons/icons/request_wait.png</normaloff>:/icons/icons/request_wait.png</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="5" rowspan="2">
      <widget class="QLabel" name="labelInterval">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>0</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateTimeEdit" name="dateTimeEditBegin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="displayFormat">
        <string>dd.MM.yyyy hh:mm:ss</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::UTC</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDateTimeEdit" name="dateTimeEditEnd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="displayFormat">
        <string>dd.MM.yyyy hh:mm:ss</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::UTC</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IntervalSlider</class>
   <extends>QWidget</extends>
   <header>intervalslider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
